ui/src/components/HookForm/index.jsx

```diff
@@ -510,52 +553,28 @@ export default class HookForm extends Component {
             <Fragment>
               <ListItem>
                 <ListItemText
-                  primary="Last Fired"
+                  primary="Next Scheduled Fire"
                   secondary={
-                    lastFireTypeName === HOOKS_LAST_FIRE_TYPE.NO_FIRE ? (
-                      'n/a'
+                    hook.status.nextScheduledDate ? (
+                      <DateDistance from={hook.status.nextScheduledDate} />
                     ) : (
-                      <DateDistance from={hook.status.lastFire.time} />
+                      'n/a'
                     )
                   }
                 />
               </ListItem>
-              {lastFireTypeName === HOOKS_LAST_FIRE_TYPE.SUCCESSFUL_FIRE ? (
-                <ListItem
-                  button
-                  component={Link}
-                  className={classes.listItemButton}
-                  to={`/tasks/${hook.status.lastFire.taskId}`}>
-                  <ListItemText
-                    primary="Last Fired Result"
-                    secondary={hook.status.lastFire.taskId}
-                  />
-                  <LinkIcon />
-                </ListItem>
-              ) : (
-                <ListItem>
-                  <ListItemText
-                    primary="Last Fired Result"
-                    secondary={
-                      lastFireTypeName === HOOKS_LAST_FIRE_TYPE.NO_FIRE ? (
-                        'n/a'
-                      ) : (
-                        <pre>
-                          {JSON.stringify(hook.status.lastFire.error, null, 2)}
-                        </pre>
-                      )
-                    }
-                  />
-                </ListItem>
-              )}
+              <ListSubheader className={classes.subheader}>
+                Last Fired Results
+              </ListSubheader>
               <ListItem>
                 <ListItemText
-                  primary="Next Scheduled Fire"
-                  secondary={
-                    hook.status.nextScheduledDate ? (
-                      <DateDistance from={hook.status.nextScheduledDate} />
-                    ) : (
-                      'n/a'
+                  primary={
+                    hookLastFires && (
+                      <HookLastFiredTable
+                        items={hookLastFires}
+                        onErrorClick={this.handleDrawerOpen}
+                        paginate
+                      />
```

Comment:
We don't really need `ListItemText`. Let's try to display the table without `ListItemText`.